index.html

```diff
@@ -8,7 +8,7 @@
         <script src="https://kit.fontawesome.com/c373984c3f.js" crossorigin="anonymous"></script>
         <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,900;1,800&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="style.css">
-        <title>My Portfolio</title>
+        <title>Belden Portfolio | Landing Page</title>
```

Comment:
- [ ] Good job Improving the title, I think it will be better if you remove the pipe `|` from the title and just use Belden's Portfolio or "Beden Full stack developer". A landing page and the pipe symbol is not correct or appropriate for accessibility, generally In digital marketing, a landing page is a standalone web page, created specifically for a marketing or advertising campaign.